spec/mongo/database_spec.rb

```diff
@@ -180,6 +180,26 @@
       it 'returns a list of the collections info' do
         expect(result).to include('acol')
       end
+
+      context 'with more than one collection' do
+        before do
+          database[:anothercol].drop
+          database[:anothercol].create
+
+          expect(database.collections.length).to be > 1
```

Comment:
@p-mongo Is this the kind of assertion you were looking for?

Reply:
Yes this works.